lib/models/opponent.dart

```diff
@@ -210,28 +210,22 @@ class Opponent {
       return;
     }
     for (int i = 0; i < GameRules.statementsPerGame; i++) {
-      // correctly found as Reak News
-      if (opponentAnswers[i] &&
-          statements.statements[i].statementCorrectness ==
-              CorrectnessCategory.correct) {
+      bool stmCorrect = CorrectnessCategory.isFact(
```

Comment:
I think for better readability I'd name this `statementCorrect`. What do you think?